src/main/java/io/snyk/jenkins/SnykStepBuilder.java

```diff
@@ -169,10 +170,11 @@ public void setAdditionalArguments(@CheckForNull String additionalArguments) {
 
   @Override
   public void perform(
-    @Nonnull Run<?, ?> build,
-    @Nonnull FilePath workspace,
-    @Nonnull Launcher launcher,
-    @Nonnull TaskListener log
+    @NonNull Run<?, ?> build,
+    @NonNull FilePath workspace,
+    @NonNull EnvVars env,
```

Comment:
Do you think we could take these environment variables to pass it on to the Context - later it is accessed in `SnykStepFlow`.

Reply:
Yes! Good catch. I addressed the deprecated `#perform` method by introducing the new `EnvVars env` parameter, but didn't do anything with it 😄 

Now it's being passed to the `SnykContext` in [49fcf8d](https://github.com/jenkinsci/snyk-security-scanner-plugin/pull/123/commits/49fcf8d641c95c431f6f3fbc9aa8c4e7916488d5), is this what you were looking for?

Reply:
Yeah, that's exactly what I had in mind :)